scripts/core/exp-command-buffer.yml

```
@@ -217,6 +217,51 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Append a USM fill command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendFillUSMExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object."
+    - type: "void*"
+      name: pMemory
+      desc: "[in] pointer to USM memory object."
```

Preference for slightly tweaked wording
```suggestion
      desc: "[in] pointer to USM allocated memory to fill."
```